opm/simulators/flow/BlackoilModelEbos.hpp

```diff
@@ -944,22 +1777,22 @@ namespace Opm {
                 double tol[2] = { tol_mb, tol_cnv };
                 for (int ii : {0, 1}) {
                     if (std::isnan(res[ii])) {
-                        report.setReservoirFailed({types[ii], CR::Severity::NotANumber, compIdx});
+                        report.setReservoirFailed({types[ii], CR::Severity::NotANumber, compIdx});//, cell[ii], res[ii]});
```

Comment:
Will we ever need `cell[ii]` and/or `res[ii]`?  If not I think we should remove those disabled quantities.

Reply:
Removed.